rest-api-spec/src/yamlRestTest/resources/rest-api-spec/test/snapshot.restore/10_basic.yml

```diff
@@ -32,6 +32,8 @@ setup:
         repository: test_repo_restore_1
         snapshot: test_snapshot
         wait_for_completion: true
+        body:
+          indices: "test_index"
```

Comment:
Alternatively I could Alternatively I could update test to expect `gre: 1` shards snapshotted at:

https://github.com/elastic/elasticsearch/blob/master/rest-api-spec/src/yamlRestTest/resources/rest-api-spec/test/snapshot.restore/10_basic.yml#L38

Reply:
Do you know if snapshot.create/10_basic.yml fails similarly? I think we can just check that the snapshot is successful with no failed shards and move on.

Reply:
There are only 2 such failures during last 2 month. Both caused by extra `".tasks"` index.

Reply:
On the other side, if I allow it to take more shards/indices I would need to modify restore part. I guess it would fail when I try to restore `.task` and it might not be trivial to close as it is system.

Reply:
What do you think of changing both tests with:
```
- match: { snapshot.snapshot: test_snapshot }
- match: { snapshot.state : SUCCESS }
# snapshot can contain system indices
- gt: { snapshot.shards.successful: 0 }
- match: { snapshot.shards.failed : 0 }
```

and add one more operation that snapshot the specific index only?